Pages/Index.cshtml

```diff
@@ -27,36 +29,36 @@
     @if (Model.IsWorkHourSet)
     {
         <div class="alert alert-secondary">
-            <p>You have set the work hours and days. <br>
-                You can modify the work hours and days by filling the form below</p>
+            <p>
+                You have set the work hours and days. <br>
+                You can modify the work hours and days by filling the form below.
+            </p>
         </div>
     }
     <hr>
 
     <p>Specify your work hours</p>
     <form class="row g-3" method="post">
         <div class="col-md-4">
-            <label for="openingTime" class="form-label">Opening Time</label>
-            <input type="time" class="form-control" name="OpeningTime" required>

```

Comment:
You have to connect the label and inputs for accessibility using `id` and `for`.